tests/testthat/test_base_fixed_indices_cv.R

```diff
@@ -0,0 +1,64 @@
+context("fixed")
+
+test_that("fixed in single resampling", {
+  df = multiclass.df
+  fixed = as.factor(rep(1:30, 5))
+  ct = makeClassifTask(target = multiclass.target, data = multiclass.df,
+    blocking = fixed)
+
+  # test blocking in single resample
+  lrn = makeLearner("classif.lda")
+  rdesc = makeResampleDesc("CV", fixed = TRUE)
+  p = resample(lrn, ct, rdesc)$pred
+
+  # check if all test.inds are unique
+  expect_length(unique(unlist(p$instance$test.inds, use.names = FALSE)), 150)
```

Comment:
Should also check whether the right observations are together.

Reply:
Done.